src/style.css

```diff
@@ -88,4 +87,4 @@ form {
   header {
     margin: 0;
   }
-} */
+}
```

Comment:
# In ReadMe File:
- Please, make sure to add instructions about where and how to generate any necessary keys, and credentials, and how to set, store, and generate them. In another word, how to run this project on a local machine(state the installations needed).